inference-engine/src/inference_engine/cpp/ie_executable_network.cpp

```diff
@@ -4,117 +4,109 @@
 
 #include "cpp/ie_executable_network.hpp"
 #include "ie_common.h"
+#include "cpp_interfaces/interface/ie_iexecutable_network_internal.hpp"
+#include "cpp_interfaces/exception2status.hpp"
+#include "ie_iexecutable_network.hpp"
+#include "cpp_interfaces/base/ie_executable_network_base.hpp"
```

Comment:
looks like `ie_executable_network_base.hpp` is used only inside `inference-engine/src/inference_engine/cpp/` and not used by plugins. Can we move to `inference-engine/src/inference_engine/cpp/` folder?